src/ontology/mondo-edit.obo

```diff
@@ -556186,67 +556195,36 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
-synonym: "early infantile epileptic encephalopathy with burst-suppression" EXACT [DOID:0050709]
-synonym: "early infantile epileptic encephalopathy with suppression-bursts" EXACT [Orphanet:1934]
-synonym: "EIEE" EXACT ABBREVIATION [NCIT:C122814, Orphanet:1934]
-synonym: "epileptic encephalopathy, early infantile" EXACT []
+synonym: "developmental and epileptic encephalopathy" BROAD [DOID:0112202, https://orcid.org/0000-0001-8486-0558, NCIT:C122814, OMIMPS:308350]
 synonym: "epileptic encephalopathy, early infantile, 15" RELATED EXCLUDE []
-synonym: "epileptic encephalopathy, infantile" EXACT []
-synonym: "infantile epileptic encephalopathy" EXACT [MONDO:0006508]
-synonym: "infantile spasm" EXACT []
-synonym: "Ohtahara syndrome" EXACT [MONDO:0000418, Orphanet:1934]
-xref: DOID:0050709 {source="MONDO:equivalentTo"}
-xref: DOID:0112202 {source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
-xref: DOID:2481 {source="MONDO:equivalentObsolete", source="EFO:1000643"}
-xref: GARD:9255 {source="MONDO:GARD"}
-xref: ICD10CM:G40.3 {source="Orphanet:1934", source="Orphanet:1934/attributed", source="Orphanet:1934/ntbt"}
-xref: ICD10CM:G40.82 {source="DOID:2481"}
+synonym: "hereditary developmental and epileptic encephalopathy" NARROW [https://orcid.org/0000-0001-5208-3432]
+xref: DOID:0112202 {source="MONDO:equivalentTo"}
 xref: ICD9:345.10 {source="MONDO:relatedTo", source="MONDO:i2s"}
-xref: ICD9:345.6 {source="DOID:2481"}
-xref: MedDRA:10071545 {source="Orphanet:1934/e", source="Orphanet:1934"}
-xref: MEDGEN:97959 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: MESH:D013036 {source="DOID:2481"}
-xref: NANDO:1200593 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0011-764X", source="https://orcid.org/0000-0002-0170-9172"}
 xref: NCIT:C122814 {source="MONDO:equivalentTo"}
-xref: NCIT:C84788 {source="MONDO:directSiblingOf", source="DOID:2481"}
 xref: OMIMPS:308350 {source="MONDO:pnr", source="MONDO:equivalentTo"}
-xref: Orphanet:1934 {source="MONDO:equivalentTo"}
-xref: SCTID:155042008 {source="DOID:2481"}
-xref: SCTID:193014002 {source="DOID:2481"}
-xref: SCTID:193015001 {source="DOID:2481"}
-xref: SCTID:193016000 {source="DOID:2481"}
-xref: SCTID:230429005 {source="MONDO:equivalentTo"}
-xref: SCTID:28055006 {source="MONDO:directSiblingOf", source="DOID:2481"}
-xref: SCTID:288197007 {source="DOID:2481"}
-xref: UMLS:C0393706 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:97959"}
-is_a: MONDO:0100022 {source="Orphanet:1934", source="https://orcid.org/0000-0001-8486-0558"} ! neonatal/infantile epilepsy syndrome
-is_a: MONDO:0100038 {source="https://orcid.org/0000-0003-1245-3907"} ! complex neurodevelopmental disorder
+is_a: MONDO:0100620 {source="https://orcid.org/0000-0001-5208-3432"} ! developmental and epileptic encephalopathy
+intersection_of: MONDO:0100620 ! developmental and epileptic encephalopathy
+intersection_of: has_characteristic MONDO:0021152 ! inherited
```

Comment:
@nicolevasilevsky, I thought this should NOT be inherited. The point the experts were saying yesterday is that they prefer "genetic" because it is not "inherited", it is most often due to de novo mutations

Reply:
So - this is going to cause a problem that is a bigger issue. This is an OMIMPS, so it must have has characteristic some inherited.

We've been using the terms inherited and genetic sort of interchangeably. I think for the purpose of the label, using genetic is fine. But we don't distinguish between inherited and genetic in the logical axioms at the moment in Mondo.

Reply:
This would need to be a bigger discussion.